spec/controllers/email_actions_controller_spec.rb

```diff
@@ -19,6 +19,24 @@
     end
   end
 
+  describe "follow_discussion" do
+    before do
+      @user = FactoryGirl.create(:user)
```

Comment:
I don't think you need to call `FactoryGirl.` here, it can just be

```
@user = create(:user)
```

Not sure why the current tests are using this format, but it's less than great form.